src/views/ManageList.jsx

```diff
@@ -1,7 +1,90 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+export function ManageList({ listPath }) {
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(0);
+
+	const handleChangeItem = (e) => {
+		setItemName(e.target.value);
+	};
+
+	const handleChangeDate = (e) => {
+		setDaysUntilNextPurchase(parseInt(e.target.value));
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const newItem = await addItem(listPath, {
+				itemName,
+				daysUntilNextPurchase,
+			});
+			alert('Item saved successfully.');
+		} catch (error) {
+			alert('Item not saved successfully.');
+		}
+		setItemName('');
+		document.getElementById('item-form').reset();
+	};
+
 	return (
-		<p>
-			Hello from the <code>/manage-list</code> page!
-		</p>
+		<div>
+			<form id="item-form" onSubmit={handleSubmit}>
+				<div>
+					<label htmlFor="item">Item Name</label>
+					<input
+						value={itemName}
+						onChange={handleChangeItem}
+						name="itemName"
+						type="text"
+						id="item"
+						required
+					/>
+				</div>
+
+				<div>
+					<label htmlFor="daysUntilNextPurchase">
+						How soon do you think you'll purchase this item again?
+					</label>
+					<ul>
+						<li>
+							<label htmlFor="soon"> Soon (7 days)</label>
+							<input
+								value={7}
+								onChange={handleChangeDate}
+								name="daysUntilNextPurchase"
+								type="radio"
+								id="soon"
```

Comment:
added `id` attribute to match `forHTML` attribute on `<label>` for accessibility